ci_tools/bot_comment_react.py

```diff
@@ -29,6 +29,9 @@ def get_unique_test_list(keys):
     tests.discard('pr_tests')
     if 'coverage' in tests:
         tests.add('linux')
+        # Ensure coverage is last in case dependencies are ready
+        tests.discard('coverage')
+        tests.add('coverage')
```

Comment:
I do not understand this change, as Python sets are not ordered AFAIK. Maybe you want to use a list instead?

Reply:
Ah. You are right, dictionaries have confused me. Fixed: a62afc4989a35e521d082320c04c3b808b65f466